mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/application/AuthenticationFacadeTest.kt

```diff
@@ -43,7 +43,59 @@ class AuthenticationFacadeTest {
 
     val facade = AuthenticationFacade(auth, firestore)
 
-    assertThat(facade.signInWithEmail("email", "password")).isEqualTo(Failure)
+    assertThat(facade.signInWithEmail("email@epfl.ch", "password")).isEqualTo(FailureInvalidUser)
+  }
+
+  @Test
+  fun Given_nonEmptyAuth_When_theEnteredPasswordIsIncorrect_Then_aFailureIncorrectPasswordOccurs() {
```

Comment:
The name doesn't respect our convention. The keyword must be in lowercase.

Reply:
Fixed in a362d0a